pom.xml

```diff
@@ -665,6 +664,13 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
```

Comment:
For reviewer: There were incompatibilities in tests due to having different versions of various jackson-modules on the classpath causing the test failures. I decided to clean this up by using the bom instead of an explicit version dependency in the parent.